certified-connectors/IN-D-Payables/apiProperties.json

```diff
@@ -15,9 +15,9 @@
         }
```

Comment:
@gayathri35 Please do not change the connector folder names frequently. Please let it be "IN-D Payables".